src/layout/HeroSection/HeroSection.svelte

```diff
@@ -41,34 +34,24 @@
 		}, 500);
 	};
 
-	const updateDownloadSource = (value: number) =>
-		localStorage.setItem("downloadSource", value.toString());
+	const updateDownloadSource = (value: number) => localStorage.setItem("downloadSource", value.toString());
 
 	const changeDownloadSource = (downloadSource: DownloadSources, id: number) => {
 		updateDownloadSource(id);
 
-		switch (downloadSource) {
-			case "Winget (CLI)":
-				wingetDialogOpen = true;
-				break;
-			case "Microsoft Store":
-			case "Sideload Package":
-			case "Sideload Package (Preview)":
-				window.open(downloadSource === "Microsoft Store" ? getStoreUrl() : sideloadLink, "_blank");
-				break;
-		}
+		if (downloadSource !== "Winget (CLI)")
+			window.open(downloadSource === "Microsoft Store" ? getStoreUrl() : sideloadLink, "_blank");
+		else wingetDialogOpen = true;
```

Comment:
nit: Use curly braces for if and else bodies.
Not nit: Don't put the else body in the same line as the else keyword.